presto-cache/src/main/java/com/facebook/presto/cache/CachingFileSystem.java

```diff
@@ -108,12 +108,24 @@ public long getDefaultBlockSize()
         return dataTier.getDefaultBlockSize();
     }
 
+    @Override
+    public long getDefaultBlockSize(Path workingDirectory)
```

Comment:
why use the workingDirectory as the name of the parameter?  I think it's just a path

Reply:
makes sense, updated